code/game/turfs/open/floor/misc_floor.dm

```diff
@@ -322,3 +322,14 @@
 
 /turf/open/floor/cult/airless
 	initial_gas_mix = AIRLESS_ATMOS
+
+/turf/open/floor/material/meat
+	desc = "living floor"
+	icon_state = "grey"
+	initial_gas_mix = AIRLESS_ATMOS
```

Comment:
would rather you have a regular meat floor version and then an airless subtype from a mapping standpoint because it's really tempting to use new turfs in new maps and people probably won't realize it's airless until it's too late- would really appreciate having the option to choose from the start